src/templates/haml/static/bulk_trader_facility.html.haml

```diff
@@ -0,0 +1,88 @@
+- stash(
+-  title => l('Bulk Trader Facility'),
+- );
+.grd-container.grd-default-container
+  .bulk-trader-facility.static_full
+    .grd-grid-3.grd-hide-mobile
+      .sidebar
+        %ul#sidebar-nav
+          %li#sidebar-link-section-1.selected
+            %a{:href => "#bulk-trader-facility"}= l('Bulk trader facility')
+          %li#sidebar-link-section-2
+            %a{:href => "#client-guide"}= l('[_1] client\'s guide', stash->{website_name})
```

Comment:
better to use double quotes here like `l("[_1] client's guide", stash->{website_name})` so there is no need to put slash in front of single quote.